### PR TITLE
Updated Vorstand page for new term

### PR DESCRIPTION
**Pull request by alanderex**
Updated after election 2023.
I used the LinkedIn pictures of Jessica and Paloma from LinkedIn, guess that'll be the right one to use. 😄

### CODE REVIEW

**Comment by kbr** [comment]
LGTM but will leave approving to Jessica/Paloma, because of the images.

**Review by sleepypioneer — COMMENTED** [review]
I'm happy with that image, thank you for checking! Will leave it for @discombobulateme to approve the PR once she checks the image :)

File: website/databags/board.json [hunk 1]
```diff
@@ -1,91 +1,94 @@
...
-          "role":" Beraterin",
-          "image": "/static/media/board/1517716762908.jpeg",
-          "twitter":"",
-          "linkedin":"",
-          "xing":""
+          "name": "Jessica Greene",
+          "role": " Vorstandsmitglied",
+          "image": "/static/media/board/jessica.jpeg",
+          "twitter": "",
+          "linkedin": "",
```
Comment (sleepypioneer) [inline website/databags/board.json:51]: do we want to add these social media links? I'm not on Xing and barely on Twitter (now X) but my LinkedIn is https://www.linkedin.com/in/jessica0greene/ maybe folks also have personal websites?

Comment (alanderex) [reply website/databags/board.json:51]: We can add it but currently it does not show on the website. I think I can add links.

**Comment by alanderex** [comment]
Happy to report social links have been added 🚀

File: website/content/blog/mv-2023/contents.lr [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+#### Jessica Greene
+
+Jessica Green hat als Quereinsteigerin eine unkonventionelle Karriere in der Technikbranche eingeschlagen, wobei die
+Community und insbesondere die Python-Community stets im Zentrum ihrer beruflichen Laufbahn standen. Sie feierte
+kürzlich ihr fünfjähriges Jubiläum als Software-Ingenieurin und ihr sechsjähriges als Mitglied der [PyLadies Berlin](https://berlin.pyladies.com/).
+Zudem engagierte sie sich im [PSF](https://psf.org) Conduct WG. Sie arbeitet Vollzeit als Entwicklerin bei [Ecosia](https://ecosia.org/) und setzt
+sich leidenschaftlich für Klimaschutz und Nachhaltigkeitsinitiativen ein. Jessica hofft, durch ihre Mitgliedschaft im
+Vorstand ihre vielfältigen Fähigkeiten und Interessen zum Wohl der Python-Community in Deutschland und der DACH-Region
+einzubringen.
```
Comment (sleepypioneer) [inline website/content/blog/mv-2023/contents.lr:107]: thanks for this @alanderex, great translation and summarisation, I might actually use this if I need a German bio in the future <3

File: website/content/blog/mv-2023/contents.lr [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+&#8291;Vorstandsmitglied: Klaus Bremer  
+&#8291;Vorstandsmitglied: Jessica Greene  
+&#8291;Vorstandsmitglied: Paloma Olivera  
+
+Es freut uns, dass der Vorstand auf nun auf sieben Mitglieder erweitert wurde und wir zwei neue
+Vorstandsmitglieder begrüßen dürfen, die wir hier kurz vorstellen:
+
+#### Jessica Greene
+
+Jessica Green hat als Quereinsteigerin eine unkonventionelle Karriere in der Technikbranche eingeschlagen, wobei die
```
Comment (sleepypioneer) [inline website/content/blog/mv-2023/contents.lr:101]:
```suggestion
Jessica Greene hat als Quereinsteigerin eine unkonventionelle Karriere in der Technikbranche eingeschlagen, wobei die
```

**Comment by sleepypioneer** [comment]
> Happy to report social links have been added 🚀
> 
> * linkedin
> * twitter
> * github
> * website
> 
> please feel free to add your in`website/databags/board.json` and make a PR
> 
> @kbr please review the post as well `website/content/blog/mv-2023/contents.lr`
> 
> I added the staging website as PR template to make it easier 😄 http://pysv-web-staging.s3-website.eu-central-1.amazonaws.com

awesome, once this merges I will add my other links. Thank you @alanderex!

File: website/content/blog/mv-2023/contents.lr [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+Es der bestehende Vorstand kandidierte wieder bis auf Christian Theune, der nach langjährigem Engagement nicht mehr
+kandidierte. Wir bedanken uns herzlich bei Christian fpr seinen langjährigen Einsatz für den Verein und die Community.
+
+Der neue [Vorstand](../../board) setzt sich wie folgt zusammen:
+
+&#8291;1. Vorsitzender: Dr.-Ing. Mike Müller  
+&#8291;2. Vorsitzender: Alexander Hendorf  
+&#8291;1. stellvertretende Vorsitzende: Dr. Tereza Iofciu  
+&#8291;2. stellvertretender Vorsitzender: Prof. Dr. Martin Christen  
+&#8291;Vorstandsmitglied: Klaus Bremer
```
Comment (kbr) [inline website/content/blog/mv-2023/contents.lr:92]: Klaus Bremer -> Dr. Klaus Bremer
(if titles, put them all in :)

File: website/content/blog/mv-2023/contents.lr [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+Konferenz zog
+1.841 Teilnehmer an.
+
+Des Weiteren veranstalteten wir als Verein die [EuroSciPy Konferenz](https://euroscipy.org/2023/) Ende August in Basel,
+die
+mehr als 200 Teilnehmer aus verschiedenen Ländern anzog. Diese Konferenz diente als Plattform für den Austausch und
+Diskussionen über
+wissenschaftliches Programmieren in Python und vereinte Akademiker, Industrievertreter und Open-Source-Maintainer.
+
+Im Jahr 2023 gingen auch mehr Förderanträge beim uns ein, und es konnten bereits über 10 Projekte finanziell oder als
```
Comment (kbr) [inline website/content/blog/mv-2023/contents.lr:43]: Typo: beim -> bei

File: website/content/blog/mv-2023/contents.lr [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+die [PyCon DE und PyData Berlin](https://2023.pycon.de) im
+bcc ausgerichtet zu haben, die als nun größte Python-Konferenz in Europa hervorstach. Ein neu formiertes, größeres
+Organisationsteam verteilte die Verantwortungsbereiche effektiv, mit einem Fokus auf Programmgestaltung, Diversität und
+Teilnehmererfahrung, während das Backoffice sich um die Eventabwicklung und das Sponsoring kümmerte. Die dreitägige
+Konferenz zog
+1.841 Teilnehmer an.
+
+Des Weiteren veranstalteten wir als Verein die [EuroSciPy Konferenz](https://euroscipy.org/2023/) Ende August in Basel,
+die
+mehr als 200 Teilnehmer aus verschiedenen Ländern anzog. Diese Konferenz diente als Plattform für den Austausch und
```
Comment (kbr) [inline website/content/blog/mv-2023/contents.lr:39]: zu der mehr als 200 Teilnehmer aus verschiedenen Ländern anreisten. (- Verdoppelung von zog und anzog)

**Review by kbr — CHANGES_REQUESTED** [review]
Still found some typos and minor improvements.

**Comment by discombobulateme** [comment]
Thank you for having all this work :) I can add or change later and create a new PR, for now its really great! :)

File: website/content/blog/mv-2023/contents.lr (hunk 3 above)
Comment (alanderex) [reply website/content/blog/mv-2023/contents.lr]: fixed, tx!

File: website/content/blog/mv-2023/contents.lr (hunk 2 above)
Comment (alanderex) [reply website/content/blog/mv-2023/contents.lr:106]: Sure, feel free to use it! 🤗

**Pull request merged**